src/test/java/nl/tudelft/contextproject/util/SizeTest.java

```diff
@@ -50,7 +49,7 @@ public void testEqualsEqual() {
 	 */
 	@Test
 	public void testEqualsSame() {
-		assertTrue(size.equals(size));
+		assertEquals(size, size);

```

Comment:
For your other question, this test is absolutely valid. An object should equal itself. This is specified in the documentation of Object.equals.